docs/dev/development.adoc

```diff
@@ -80,19 +80,20 @@ $ git remote set-url --push upstream no_push
 
 === About Prow
 `odo` uses the link:https://github.com/kubernetes/test-infra/tree/master/prow[Prow] infrastucture for CI testing.
+
 * It uses link:https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md[OWNERS] files to determine who can approve and lgtm a PR.
 * Prow has two levels of OWNERS, *Approvers* and *Reviewers*
 ** *Approvers* look for holistic acceptance criteria, including
 dependencies with other features, forward and backward compatibility, API and flag definitions, etc. In essence, the high levels of design
 ** *Reviewers* look for general code quality, correctness, sane software engineering, style, etc. In essence, the quality of the actual code itself.
 
-* Avoid merging the PR manually, unless it is an emergency and  you have the required permissions). Prow’s tide component automatically merges the PR once all the conditions are met.
+* Avoid merging the PR manually (unless it is an emergency and  you have the required permissions). Prow’s tide component automatically merges the PR once all the conditions are met.
 It also ensures that post-submit tests (tests that run before merge) validate the PR.
 * Use the link:https://deck-ci.svc.ci.openshift.org/command-help[command-help] to see the list of possible bot commands.
 
-== Test Driven Development
+== Tests
 
-We follow the Test Driven Development(TDD) workflow in our development process. You can read more about it link:tdd-workflow.adoc[here].
+We use both unit, integration and e2e (End-2-End) tests.
```

Comment:
Aren't those three kind of tests? Referring to the word `both` being used in that statement.